src/pages/serverError/serverError.css

```
@@ -0,0 +1,29 @@
+@import url(../../styles.css);
+
+.content {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.content__title {
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 40px;
+  letter-spacing: 0em;
+  text-align: center;
+  margin: 0 0 20px 0;
```

Такие моменты тоже сокращаются
```suggestion
  margin: 0 0 20px;
```

https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties